src/ocean.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="node" />
+
+import * as path from 'path';
+import * as fs from 'fs';
+import * as turf from '@turf/turf';
+import {Polygon, FeatureCollection} from 'geojson';
+
+const data: FeatureCollection<Polygon> = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../data/oceans.json'), {encoding: 'utf-8'}));
```

Comment:
Would it be possible to just use the CommonJS `require` to pull the JSON in with `const data: FeatureCollection<Polygon> = require('../data/oceans.json')`?